HISTORY.rst

```diff
@@ -22,6 +22,7 @@ XXXX-XX-XX
 - 2542_: if system clock is updated `Process.children()`_ and
   `Process.parent()`_ may not be able to return the right information.
 - 2545_: [illumos]: Fix handling of MIB2_UDP_ENTRY in `net_connections()`_.
+- 2547_: [illumos]: Add "unix" to conn_tmap on illumos/SunOS too.
```

Comment:
What's the change exactly? Was only the unit test broken or also the function per se? If the former, please explain what was the problem (exception? crash?).

Reply:
The change exactly is to add "unix" back to `conn_tmap` as it was before December 2024.

I found the issue by running `psutil`'s tests.  I do not use `psutil` for anything specific.  I just package it for OpenIndiana (and so I run its tests to make sure it works), primarily because it is needed by some other Python projects packaged.

Reply:
It looks like UNIX sockets are not supported on SunOS, that's why I removed it:
https://github.com/giampaolo/psutil/blob/11fd4026d55b978b95ac6d15398b85c6c0c52bec/psutil/_pssunos.py#L262